lib/core/src/lib/card-view/components/card-view-textitem/card-view-textitem.component.scss

```diff
@@ -1,204 +1,57 @@
-        font-size: var(--theme-subheading-2-font-size);
-        width: 16px;
-        height: 16px;
-        color: var(--adf-theme-foreground-text-color-025);
+.adf-card-view-textitem {
+    .adf-textitem-error {
+        font-size: var(--theme-caption-font-size);
+        padding-top: 6px;
+
+        ul {
+            margin: 0;
+            padding: 0;
+            list-style-type: none;
+
+            li {
+                margin: 0;
+                padding: 0;
+            }
+        }
     }
 
-    &-textitem-action.mat-icon-button {
+    .adf-textitem-action {
         width: 20px;
         height: 20px;
         line-height: 20px;
         color: var(--adf-theme-foreground-text-color-025);
-    }
-
-    &-textitem-action:hover,
-    &-textitem-action:focus {
-        color: var(--adf-theme-foreground-text-color);
-    }
-
-    &-update-icon {
-        padding-left: 13px;
-    }
-
-    &-textitem-readonly {
-        cursor: pointer !important;
 
-        &:hover .adf-textitem-action,
-        &:focus .adf-textitem-action {
+        &:hover,
+        &:focus {
             color: var(--adf-theme-foreground-text-color);
         }
     }
 
-    &-textitem-chip-list-container {
-        margin-bottom: 25px !important;
-        margin-top: 6px;
-
+    .adf-textitem-chip-list-container {
         .mat-form-field-label {
             margin-top: 6px;
         }
     }
 
-    &-textitem-clickable {
+    .adf-textitem-clickable {
         cursor: pointer !important;
         padding-top: 3px;
 
-        .adf-textitem-edit-icon.mat-icon {
-            line-height: 16px;
-        }
-
-        &:hover .adf-textitem-action {
+        .adf-textitem-action:hover {
             color: var(--adf-theme-foreground-text-color);
         }
-    }
 
-    &-textitem-clickable-value {
-        cursor: pointer !important;
-        color: var(--theme-primary-color) !important;
-    }
-
-    &-textitem-editable {
-        &-controls {
-            display: flex;
-            align-items: center;
-
-            mat-icon:not(.adf-button-disabled):hover {
-                color: var(--adf-theme-foreground-text-color);
-                cursor: pointer !important;
-            }
-
-            mat-form-field {
-                width: 100%;
-            }
-
-            input:focus,
-            textarea:focus {
-                border: 1px solid var(--theme-accent-color-a200);
-            }
-        }
-
-        &-error {
-            font-size: var(--theme-caption-font-size);
-            padding-top: 6px;
-
-            ul {
-                margin: 0;
-                padding: 0;
-                list-style-type: none;
-
-                li {
-                    margin: 0;
-                    padding: 0;
-                }
-            }
+        &-value {
+            cursor: pointer !important;
+            color: var(--theme-primary-color) !important;
         }
     }
 
-    &-textitem-default-value {
+    .adf-textitem-default-value {
```

Comment:
should not be text-item instead of textitem?

Reply:
the component is called textitem, so the style resembles the naming, changing the name might cause extra troubles